libs/libxslt/manifest.x86_64.jsonc

```diff
@@ -0,0 +1,163 @@
+/** Human readable report. This is not consumed by boulder */
+{
+	"manifest-version": "0.2",
+	"packages": {
+		"libxslt": {
+			"build-depends": [
+				"pkgconfig(libxml-2.0)",
+				"pkgconfig(python3)"
+			],
+			"depends": [
+				"interpreter(/usr/lib/ld-linux-x86-64.so.2(x86_64))",
+				"soname(libc.so.6(x86_64))",
+				"soname(libm.so.6(x86_64))",
+				"soname(libxml2.so.2(x86_64))"
+			],
+			"files": [
+				"/usr/bin/xslt-config",
+				"/usr/bin/xsltproc",
+				"/usr/lib/libexslt.so.0",
+				"/usr/lib/libexslt.so.0.8.20",
+				"/usr/lib/libxslt-plugins",
+				"/usr/lib/libxslt.so.1",
+				"/usr/lib/libxslt.so.1.1.37",
+				"/usr/lib/python3.11/site-packages/__pycache__/libxslt.cpython-311.opt-1.pyc",
+				"/usr/lib/python3.11/site-packages/__pycache__/libxslt.cpython-311.pyc",
+				"/usr/lib/python3.11/site-packages/libxslt.py",
+				"/usr/lib/python3.11/site-packages/libxsltmod.so",
```

Comment:
What do you think about splitting these out to a `python-libxslt` package?

Reply:
Already saw that at the brotli package, I'm fine with that